tests/tests.toml

```diff
@@ -827,6 +831,10 @@ asr = true
 filename = "errors/test_list_slicing.py"
 asr = true
 
+[[test]]
+filename = "errors/test_list_const.py"
+run = true
```

Comment:
```suggestion
asr = true
```

Reply:
Hmm..I am a bit confused with this @Shaikh-Ubaid. I request you to please explain me when we use what.

Reply:
We use `run` when you want to store the output generated when running the test case after compilation. We use `asr` when we want to store the `ASR` output during compilation.

In this case, since we want to track the generation of a `SemanticError()` (which is present at the ASR level), we just need the test case to be compiled to ASR.